src/cljc/rems/common/util.cljc

```diff
@@ -7,6 +7,19 @@
 ;; regex from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#Validation
 (def +email-regex+ #"[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*")
 
+;; regex from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#Validation
```

Comment:
is this comment true or just a copypaste error?

Reply:
copypaster error